templates/chain.html

```diff
@@ -302,6 +303,36 @@ <h4 id="name"></h4>
             <br>
         </li>
 
+        <div id="planners" class="section-profile planners" style="display: none">
+          <div class="row">
+            <div class="column section-border" style="flex:25%">
+              <img src="/gui/img/planners.png">
+              <h4>Planners</h4>
+              <p class="section-description">
+                  A planner is a module within CALDERA which contains logic for how a running operation should make decisions about
+                  which abilities to use and in what order.
+
+                  Specifically, a planner's logic contains the decision making to execute a single phase of an operation.
+              </p>
+              <div class="topleft duk-icon"><img onclick="removeSection('planners')" src="/gui/img/x.png"></div>
+              <!-- <div class="topright duk-icon"><img onclick="openDuk2()" src="/gui/img/duk.png"></div> -->
```

Comment:
remove this comment if unused

Reply:
on second thought, we should uncomment it and make the pop up describe the process for creating a new planner.

Reply:
@privateducky As per discussion, I'll hold off on the icon until we have some good documentation for creating a new planner.